### PR TITLE
Reconciliation

### PR DESCRIPTION
**Pull request by NS4nti**
Add pubsub mechanism for the reconciliation job

### CODE REVIEW

File: vulture_os/gui/crontab/reconcile.py [hunk 1]
```diff
@@ -42,36 +42,36 @@
 # Mongo import
 from toolkit.mongodb.mongo_base import MongoBase
 
+REDIS_LIST = "darwin_alerts"
+REDIS_CHANNEL = "darwin.alerts"
+REDIS_RECONCILIED_CHANNEL = "vlt.darwin.alerts"
```
Comment (Raznak) [inline vulture_os/gui/crontab/reconcile.py]: Move this declaration to the Django settings.py

**Comment by Raznak** [comment]
This script is no longer a crontab. We must move it to the **daemon** folder.

File: vulture_os/gui/crontab/reconcile.py (hunk 1 above)
Comment (NS4nti) [reply vulture_os/gui/crontab/reconcile.py]: done :)

**Comment by NS4nti** [comment]
> This script is no longer a crontab. We must move it to the **daemon** folder.

done !

File: vulture_os/daemons/reconcile.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        while (alert is not None) and (not self.shutdown_flag.is_set()):
+            alert = alert.decode()
+            alert_handler(alert, m, r)
+            alert = r.redis.rpop(redis_list_name)
+
+    def reconcile(self):
+        m = MongoBase()
+        if not m.connect():
+            return False
+        m.connect_primary()
```
Comment (NS4nti) [inline vulture_os/daemons/reconcile.py]: import system cluster models

cluster.current node 
is master mongo ? 
return False

File: vulture_os/daemons/reconcile.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        while (alert is not None) and (not self.shutdown_flag.is_set()):
+            alert = alert.decode()
+            alert_handler(alert, m, r)
+            alert = r.redis.rpop(redis_list_name)
+
+    def reconcile(self):
+        node = Cluster.get_current_node()
+        if not node.is_master_mongo: 
+            return False
+
```
Comment (NS4nti) [inline vulture_os/daemons/reconcile.py]: @Raznak

**Pull request merged**